version.go

```diff
@@ -18,60 +18,60 @@ var (
 	//
 	// The expected process for setting the version in releases is as follows:
 	// - Create a release branch of the form 'release-vMAJOR.MINOR'
-	// - Modify the Version variable below on that branch to:
+	// - Modify the version variable below on that branch to:
 	//   - Remove the pre-release portion
 	//   - Set the build metadata to 'release.local'
-	// - Update the Version variable below on the master branch to the next
+	// - Update the version variable below on the master branch to the next
 	//   expected version while retaining a pre-release of 'pre'
 	//
 	// These steps ensure that building from source produces versions that are
-	// distinct from reproducible builds that override the Version via linker
+	// distinct from reproducible builds that override the version via linker
 	// flags.
 
-	// Version is the application version per the semantic versioning 2.0.0 spec
+	// version is the application version per the semantic versioning 2.0.0 spec
 	// (https://semver.org/).
 	//
 	// It is defined as a variable so it can be overridden during the build
 	// process with:
-	// '-ldflags "-X main.Version=fullsemver"'
+	// '-ldflags "-X main.version=fullsemver"'
 	// if needed.
 	//
 	// It MUST be a full semantic version per the semantic versioning spec or
 	// the app will panic at runtime.  Of particular note is the pre-release
 	// and build metadata portions MUST only contain characters from
 	// semanticAlphabet.
-	Version = "1.3.0-pre"
+	version = "1.3.0-pre"
```

Comment:
These should actually remain exported, because it allows the reproducible build tooling to override things for release builds.

Reply:
I tested this before opening the PR and it still works.

```
$ go build -ldflags "-X main.version=9.9.9" . && ./dcrpool --version
dcrpool version 9.9.9+e63d74502 (Go version go1.22.5 linux/amd64)
```

Reply:
Hmm, ok.  I wonder if that changed at some point.  I know it didn't work back when the pattern was first implemented.